params/default.go

```diff
@@ -14,6 +15,51 @@ func DefaultNetwork() Network {
 }
 
 func init() {
+	// check if custom network option set
+	if customNet, ok := os.LookupEnv("CELESTIA_CUSTOM_GENESIS"); ok {
+		fmt.Print("\n\nWARNING: Celestia custom network specified. Only use this option if the node is " +
+			"freshly created and initialized.\n**DO NOT** run a custom network over an already-existing node " +
+			"store!\n\n")
+
+		params := strings.Split(customNet, "=")
+		// ensure both params are present
+		if len(params) != 2 {
+			panic("must provide CELESTIA_CUSTOM_GENESIS in this format: <network_ID>=<genesis hash>")
+		}
+		netID, genHash := params[0], params[1]
+
+		// register new network and set as default for node to use
+		networksList[Network(netID)] = struct{}{}
+		genesisList[defaultNetwork] = strings.ToUpper(genHash)
+	}
+	// check if a custom network was specified
+	if network, ok := os.LookupEnv("CELESTIA_CUSTOM_NETWORK"); ok {
+		if err := Network(network).Validate(); err != nil {
+			panic("unknown network specified")
+		}
+		defaultNetwork = Network(network)
```

Comment:
Hmmm. This is a bit weird because you construct `Network(network)` twice. Maybe refactor?
```suggestion
		n = Network(network)
		if err := n.Validate(); err != nil {
			panic("unknown network specified")
		}
		defaultNetwork = n
```

Reply:
w/ new changes (https://github.com/celestiaorg/celestia-node/pull/717/commits/c2ac8d229a329b9d6ef592cadc7e4315f55dd059), validation on network seems redundant. `Validate` gets called elsewhere on start-up anyway. All it does is check that network exists in map and we add it to map anyway.